app/uk/gov/hmrc/homeofficesettledstatusstubs/stubdata/TestStubData.scala

```diff
@@ -779,6 +844,8 @@ object TestStubData extends DataSet {
     (PROTECTION2, "RN207514D", "BRP", "ABCDE8888"),
     (PROTECTION3, "RJ485028C", "BRP", "ABCDEF888"),
     (PROTECTION4, "MJ738984A", "BRP", "ABC88888"),
+    (frankieHolliwood, "CM878654C", "BRC", "ABC88888"),
```

Comment:
with the MRZ keys being identical this may cause an issue, assume your only using this via nino?

Reply:
They're not identical